xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/TransactionResultCode.java

```diff
@@ -0,0 +1,146 @@
+package org.xrpl.xrpl4j.model.transactions;
+
+/**
+ * The rippled server summarizes transaction results with result codes, which appear
+ * in fields such as engine_result and meta.TransactionResult.
+ */
+public class TransactionResultCode {
```

Comment:
Small nit, but can we rename this to `TransactionResultCodes`? Usually when you have a class that just contains a bunch of static methods, the class name is plural. The Guava library does this a lot, like `Strings`, `Lists`, `Maps` etc